src/screens/ChannelListScreen/elements/Search.js

```diff
@@ -43,8 +39,8 @@ const Search = ({onPress, animatedValue, isAnon, isShowNewChat = true}) => {
           </Text>
           <View>
             <MemoIcNewChat
-              height={17}
-              width={15}
+              height={18}
+              width={16}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The SVG icon size has been changed. Please ensure that these changes do not distort the icon or make it less visible.

```diff
- height={17}
- width={15}
+ height={18}
+ width={16}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->